src/legacy/ui/public/vis/vis_factory.js

```diff
@@ -17,24 +17,26 @@
  * under the License.
  */
 
-import { BaseVisTypeProvider, AngularVisTypeProvider, ReactVisTypeProvider, VislibVisTypeProvider } from './vis_types';
+import { BaseVisType, AngularVisTypeProvider, ReactVisType, VislibVisTypeProvider } from './vis_types';
+
+export const visFactory = {
+  createBaseVisualization: (config) => {
```

Comment:
You moved only these two here, because angular and vislib are to be deprecated, right?
Maybe add a comment for this

Reply:
that + the other 2 actually have an angular dependency.